[css-typed-om] remove ENABLED(CSS_TYPED_OM) compile-time flag

graouts opened the pull request:
#### 6ac3761da1f2fde724e414714f560480b113f04d
<pre>
[css-typed-om] remove ENABLED(CSS_TYPED_OM) compile-time flag
<a href="https://bugs.webkit.org/show_bug.cgi?id=246884">https://bugs.webkit.org/show_bug.cgi?id=246884</a>

Reviewed by Chris Dumez.

* Source/WTF/Scripts/Preferences/WebPreferencesExperimental.yaml:
* Source/WTF/wtf/PlatformEnableCocoa.h:
* Source/WebCore/Modules/webcodecs/WebCodecsVideoFrame.cpp:
(WebCore::checkImageUsability):
(WebCore::WebCodecsVideoFrame::create):
* Source/WebCore/Modules/webcodecs/WebCodecsVideoFrame.h:
* Source/WebCore/Modules/webcodecs/WebCodecsVideoFrame.idl:
* Source/WebCore/bindings/js/JSCSSStyleValueCustom.cpp:
* Source/WebCore/bindings/js/JSCSSTransformComponentCustom.cpp:
* Source/WebCore/css/DOMCSSNamespace+CSSNumericFactory.idl:
* Source/WebCore/css/ElementCSSInlineStyle.idl:
* Source/WebCore/css/typedom/CSSKeywordValue.cpp:
* Source/WebCore/css/typedom/CSSKeywordValue.h:
* Source/WebCore/css/typedom/CSSKeywordValue.idl:
* Source/WebCore/css/typedom/CSSNumericFactory.cpp:
* Source/WebCore/css/typedom/CSSNumericFactory.h:
* Source/WebCore/css/typedom/CSSNumericValue.cpp:
* Source/WebCore/css/typedom/CSSNumericValue.h:
* Source/WebCore/css/typedom/CSSNumericValue.idl:
* Source/WebCore/css/typedom/CSSOMVariableReferenceValue.cpp:
* Source/WebCore/css/typedom/CSSOMVariableReferenceValue.h:
* Source/WebCore/css/typedom/CSSOMVariableReferenceValue.idl:
* Source/WebCore/css/typedom/CSSStyleImageValue.cpp:
* Source/WebCore/css/typedom/CSSStyleImageValue.h:
* Source/WebCore/css/typedom/CSSStyleImageValue.idl:
* Source/WebCore/css/typedom/CSSStyleValue.cpp:
* Source/WebCore/css/typedom/CSSStyleValue.h:
* Source/WebCore/css/typedom/CSSStyleValue.idl:
* Source/WebCore/css/typedom/CSSStyleValueFactory.cpp:
* Source/WebCore/css/typedom/CSSStyleValueFactory.h:
* Source/WebCore/css/typedom/CSSUnitValue.cpp:
* Source/WebCore/css/typedom/CSSUnitValue.h:
* Source/WebCore/css/typedom/CSSUnitValue.idl:
* Source/WebCore/css/typedom/CSSUnparsedValue.cpp:
* Source/WebCore/css/typedom/CSSUnparsedValue.h:
* Source/WebCore/css/typedom/CSSUnparsedValue.idl:
* Source/WebCore/css/typedom/ComputedStylePropertyMapReadOnly.cpp:
* Source/WebCore/css/typedom/ComputedStylePropertyMapReadOnly.h:
* Source/WebCore/css/typedom/StylePropertyMap.h:
* Source/WebCore/css/typedom/StylePropertyMap.idl:
* Source/WebCore/css/typedom/StylePropertyMapReadOnly.cpp:
* Source/WebCore/css/typedom/StylePropertyMapReadOnly.h:
* Source/WebCore/css/typedom/StylePropertyMapReadOnly.idl:
* Source/WebCore/css/typedom/color/CSSColor.cpp:
* Source/WebCore/css/typedom/color/CSSColor.h:
* Source/WebCore/css/typedom/color/CSSColor.idl:
* Source/WebCore/css/typedom/color/CSSColorValue.h:
* Source/WebCore/css/typedom/color/CSSColorValue.idl:
* Source/WebCore/css/typedom/color/CSSHSL.cpp:
* Source/WebCore/css/typedom/color/CSSHSL.h:
* Source/WebCore/css/typedom/color/CSSHSL.idl:
* Source/WebCore/css/typedom/color/CSSHWB.cpp:
* Source/WebCore/css/typedom/color/CSSHWB.h:
* Source/WebCore/css/typedom/color/CSSHWB.idl:
* Source/WebCore/css/typedom/color/CSSLCH.cpp:
* Source/WebCore/css/typedom/color/CSSLCH.h:
* Source/WebCore/css/typedom/color/CSSLCH.idl:
* Source/WebCore/css/typedom/color/CSSLab.cpp:
* Source/WebCore/css/typedom/color/CSSLab.h:
* Source/WebCore/css/typedom/color/CSSLab.idl:
* Source/WebCore/css/typedom/color/CSSOKLCH.cpp:
* Source/WebCore/css/typedom/color/CSSOKLCH.h:
* Source/WebCore/css/typedom/color/CSSOKLCH.idl:
* Source/WebCore/css/typedom/color/CSSOKLab.cpp:
* Source/WebCore/css/typedom/color/CSSOKLab.h:
* Source/WebCore/css/typedom/color/CSSOKLab.idl:
* Source/WebCore/css/typedom/color/CSSRGB.cpp:
* Source/WebCore/css/typedom/color/CSSRGB.h:
* Source/WebCore/css/typedom/color/CSSRGB.idl:
* Source/WebCore/css/typedom/numeric/CSSMathClamp.cpp:
* Source/WebCore/css/typedom/numeric/CSSMathClamp.h:
* Source/WebCore/css/typedom/numeric/CSSMathClamp.idl:
* Source/WebCore/css/typedom/numeric/CSSMathInvert.cpp:
* Source/WebCore/css/typedom/numeric/CSSMathInvert.h:
* Source/WebCore/css/typedom/numeric/CSSMathInvert.idl:
* Source/WebCore/css/typedom/numeric/CSSMathMax.cpp:
* Source/WebCore/css/typedom/numeric/CSSMathMax.h:
* Source/WebCore/css/typedom/numeric/CSSMathMax.idl:
* Source/WebCore/css/typedom/numeric/CSSMathMin.cpp:
* Source/WebCore/css/typedom/numeric/CSSMathMin.h:
* Source/WebCore/css/typedom/numeric/CSSMathMin.idl:
* Source/WebCore/css/typedom/numeric/CSSMathNegate.cpp:
* Source/WebCore/css/typedom/numeric/CSSMathNegate.h:
* Source/WebCore/css/typedom/numeric/CSSMathNegate.idl:
* Source/WebCore/css/typedom/numeric/CSSMathOperator.h:
* Source/WebCore/css/typedom/numeric/CSSMathOperator.idl:
* Source/WebCore/css/typedom/numeric/CSSMathProduct.cpp:
* Source/WebCore/css/typedom/numeric/CSSMathProduct.h:
* Source/WebCore/css/typedom/numeric/CSSMathProduct.idl:
* Source/WebCore/css/typedom/numeric/CSSMathSum.cpp:
* Source/WebCore/css/typedom/numeric/CSSMathSum.h:
* Source/WebCore/css/typedom/numeric/CSSMathSum.idl:
* Source/WebCore/css/typedom/numeric/CSSMathValue.h:
* Source/WebCore/css/typedom/numeric/CSSMathValue.idl:
* Source/WebCore/css/typedom/numeric/CSSNumericArray.cpp:
* Source/WebCore/css/typedom/numeric/CSSNumericArray.h:
* Source/WebCore/css/typedom/numeric/CSSNumericArray.idl:
* Source/WebCore/css/typedom/numeric/CSSNumericBaseType.h:
* Source/WebCore/css/typedom/numeric/CSSNumericBaseType.idl:
* Source/WebCore/css/typedom/numeric/CSSNumericType.cpp:
* Source/WebCore/css/typedom/numeric/CSSNumericType.h:
* Source/WebCore/css/typedom/numeric/CSSNumericType.idl:
* Source/WebCore/css/typedom/transform/CSSMatrixComponent.cpp:
* Source/WebCore/css/typedom/transform/CSSMatrixComponent.h:
* Source/WebCore/css/typedom/transform/CSSMatrixComponent.idl:
* Source/WebCore/css/typedom/transform/CSSMatrixComponentOptions.h:
* Source/WebCore/css/typedom/transform/CSSMatrixComponentOptions.idl:
* Source/WebCore/css/typedom/transform/CSSPerspective.cpp:
* Source/WebCore/css/typedom/transform/CSSPerspective.h:
* Source/WebCore/css/typedom/transform/CSSPerspective.idl:
* Source/WebCore/css/typedom/transform/CSSRotate.cpp:
* Source/WebCore/css/typedom/transform/CSSRotate.h:
* Source/WebCore/css/typedom/transform/CSSRotate.idl:
* Source/WebCore/css/typedom/transform/CSSScale.cpp:
* Source/WebCore/css/typedom/transform/CSSScale.h:
* Source/WebCore/css/typedom/transform/CSSScale.idl:
* Source/WebCore/css/typedom/transform/CSSSkew.cpp:
* Source/WebCore/css/typedom/transform/CSSSkew.h:
* Source/WebCore/css/typedom/transform/CSSSkew.idl:
* Source/WebCore/css/typedom/transform/CSSSkewX.cpp:
* Source/WebCore/css/typedom/transform/CSSSkewX.h:
* Source/WebCore/css/typedom/transform/CSSSkewX.idl:
* Source/WebCore/css/typedom/transform/CSSSkewY.cpp:
* Source/WebCore/css/typedom/transform/CSSSkewY.h:
* Source/WebCore/css/typedom/transform/CSSSkewY.idl:
* Source/WebCore/css/typedom/transform/CSSTransformComponent.cpp:
* Source/WebCore/css/typedom/transform/CSSTransformComponent.h:
* Source/WebCore/css/typedom/transform/CSSTransformComponent.idl:
* Source/WebCore/css/typedom/transform/CSSTransformValue.cpp:
* Source/WebCore/css/typedom/transform/CSSTransformValue.h:
* Source/WebCore/css/typedom/transform/CSSTransformValue.idl:
* Source/WebCore/css/typedom/transform/CSSTranslate.cpp:
* Source/WebCore/css/typedom/transform/CSSTranslate.h:
* Source/WebCore/css/typedom/transform/CSSTranslate.idl:
* Source/WebCore/dom/Element+ComputedStyleMap.idl:
* Source/WebCore/dom/Element.cpp:
(WebCore::Element::~Element):
* Source/WebCore/dom/Element.h:
* Source/WebCore/dom/ElementRareData.cpp:
* Source/WebCore/dom/ElementRareData.h:
(WebCore::ElementRareData::setComputedStyleMap):
(WebCore::ElementRareData::useTypes const):
* Source/WebCore/dom/StyledElement.cpp:
(WebCore::StyledElement::ensureAttributeStyleMap):
* Source/WebCore/dom/StyledElement.h:
* Source/WebCore/html/ImageBitmap.cpp:
(WebCore::ImageBitmap::createPromise):
* Source/WebCore/html/ImageBitmap.h:
* Source/WebCore/html/canvas/CanvasDrawImage.idl:
* Source/WebCore/html/canvas/CanvasFillStrokeStyles.idl:
* Source/WebCore/html/canvas/CanvasRenderingContext2DBase.cpp:
(WebCore::size):
(WebCore::CanvasRenderingContext2DBase::drawImage):
(WebCore::CanvasRenderingContext2DBase::createPattern):
* Source/WebCore/html/canvas/CanvasRenderingContext2DBase.h:
* Source/WebCore/inspector/InspectorCanvas.cpp:
(WebCore::InspectorCanvas::indexForData):
* Source/WebCore/inspector/InspectorCanvas.h:
* Source/WebCore/inspector/InspectorCanvasCallTracer.h:
* Source/WebCore/page/DeprecatedGlobalSettings.h:
(WebCore::DeprecatedGlobalSettings::cssTypedOMEnabled):
* Source/WebCore/page/WindowOrWorkerGlobalScope.idl:
* Source/WebKit/WebProcess/Inspector/WebInspectorUI.cpp:
(WebKit::WebInspectorUI::enableFrontendFeatures):
* Source/cmake/OptionsGTK.cmake:
* Source/cmake/OptionsMac.cmake:
* Source/cmake/OptionsPlayStation.cmake:
* Source/cmake/OptionsWPE.cmake:
* Source/cmake/OptionsWin.cmake:
* Source/cmake/WebKitFeatures.cmake:

Canonical link: <a href="https://commits.webkit.org/255862@main">https://commits.webkit.org/255862@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/1d86aa9e1dfb8391f46b7386e201f23cc45fc377

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/93795 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/2990 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/24381 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/103435 "Built successfully") | [✅ 🛠 🧪 win](https://ews-build.webkit.org/#/builders/10/builds/163758 "Built successfully and passed tests") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/11/builds/97788 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/3006 "Built successfully") | [✅ 🛠 mac-debug](https://ews-build.webkit.org/#/builders/71/builds/31236 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/86126 "Built successfully") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/12/builds/99448 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/99455 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/2145 "Passed tests") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/80229 "Built successfully") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/29171 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/84071 "Passed tests") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/3/builds/83781 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/72126 "Passed tests") | 
| [✅ 🛠 🧪 jsc](https://ews-build.webkit.org/#/builders/1/builds/85051 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/66/builds/37631 "Built successfully") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/17609 "Passed tests") | [✅ 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/38/builds/80181 "Built successfully") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/67/builds/35487 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/18869 "Passed tests") | [✅ 🧪 jsc-armv7-tests](https://ews-build.webkit.org/#/builders/46/builds/27660 "Passed tests") | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/63/builds/39365 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/60/builds/41441 "Passed tests") | [✅ 🛠 jsc-mips](https://ews-build.webkit.org/#/builders/37/builds/82824 "Built successfully") | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/1905 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/65/builds/41298 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/38100 "Passed tests") | [✅ 🧪 jsc-mips-tests](https://ews-build.webkit.org/#/builders/45/builds/18718 "Passed tests") | 
<!--EWS-Status-Bubble-End-->

cdumez reviewed (approved):
LGTM if the bots are happy.

graouts committed:
[css-typed-om] remove ENABLED(CSS_TYPED_OM) compile-time flag
https://bugs.webkit.org/show_bug.cgi?id=246884

Reviewed by Chris Dumez.

* Source/WTF/Scripts/Preferences/WebPreferencesExperimental.yaml:
* Source/WTF/wtf/PlatformEnableCocoa.h:
* Source/WebCore/Modules/webcodecs/WebCodecsVideoFrame.cpp:
(WebCore::checkImageUsability):
(WebCore::WebCodecsVideoFrame::create):
* Source/WebCore/Modules/webcodecs/WebCodecsVideoFrame.h:
* Source/WebCore/Modules/webcodecs/WebCodecsVideoFrame.idl:
* Source/WebCore/bindings/js/JSCSSStyleValueCustom.cpp:
* Source/WebCore/bindings/js/JSCSSTransformComponentCustom.cpp:
* Source/WebCore/css/DOMCSSNamespace+CSSNumericFactory.idl:
* Source/WebCore/css/ElementCSSInlineStyle.idl:
* Source/WebCore/css/typedom/CSSKeywordValue.cpp:
* Source/WebCore/css/typedom/CSSKeywordValue.h:
* Source/WebCore/css/typedom/CSSKeywordValue.idl:
* Source/WebCore/css/typedom/CSSNumericFactory.cpp:
* Source/WebCore/css/typedom/CSSNumericFactory.h:
* Source/WebCore/css/typedom/CSSNumericValue.cpp:
* Source/WebCore/css/typedom/CSSNumericValue.h:
* Source/WebCore/css/typedom/CSSNumericValue.idl:
* Source/WebCore/css/typedom/CSSOMVariableReferenceValue.cpp:
* Source/WebCore/css/typedom/CSSOMVariableReferenceValue.h:
* Source/WebCore/css/typedom/CSSOMVariableReferenceValue.idl:
* Source/WebCore/css/typedom/CSSStyleImageValue.cpp:
* Source/WebCore/css/typedom/CSSStyleImageValue.h:
* Source/WebCore/css/typedom/CSSStyleImageValue.idl:
* Source/WebCore/css/typedom/CSSStyleValue.cpp:
* Source/WebCore/css/typedom/CSSStyleValue.h:
* Source/WebCore/css/typedom/CSSStyleValue.idl:
* Source/WebCore/css/typedom/CSSStyleValueFactory.cpp:
* Source/WebCore/css/typedom/CSSStyleValueFactory.h:
* Source/WebCore/css/typedom/CSSUnitValue.cpp:
* Source/WebCore/css/typedom/CSSUnitValue.h:
* Source/WebCore/css/typedom/CSSUnitValue.idl:
* Source/WebCore/css/typedom/CSSUnparsedValue.cpp:
* Source/WebCore/css/typedom/CSSUnparsedValue.h:
* Source/WebCore/css/typedom/CSSUnparsedValue.idl:
* Source/WebCore/css/typedom/ComputedStylePropertyMapReadOnly.cpp:
* Source/WebCore/css/typedom/ComputedStylePropertyMapReadOnly.h:
* Source/WebCore/css/typedom/StylePropertyMap.h:
* Source/WebCore/css/typedom/StylePropertyMap.idl:
* Source/WebCore/css/typedom/StylePropertyMapReadOnly.cpp:
* Source/WebCore/css/typedom/StylePropertyMapReadOnly.h:
* Source/WebCore/css/typedom/StylePropertyMapReadOnly.idl:
* Source/WebCore/css/typedom/color/CSSColor.cpp:
* Source/WebCore/css/typedom/color/CSSColor.h:
* Source/WebCore/css/typedom/color/CSSColor.idl:
* Source/WebCore/css/typedom/color/CSSColorValue.h:
* Source/WebCore/css/typedom/color/CSSColorValue.idl:
* Source/WebCore/css/typedom/color/CSSHSL.cpp:
* Source/WebCore/css/typedom/color/CSSHSL.h:
* Source/WebCore/css/typedom/color/CSSHSL.idl:
* Source/WebCore/css/typedom/color/CSSHWB.cpp:
* Source/WebCore/css/typedom/color/CSSHWB.h:
* Source/WebCore/css/typedom/color/CSSHWB.idl:
* Source/WebCore/css/typedom/color/CSSLCH.cpp:
* Source/WebCore/css/typedom/color/CSSLCH.h:
* Source/WebCore/css/typedom/color/CSSLCH.idl:
* Source/WebCore/css/typedom/color/CSSLab.cpp:
* Source/WebCore/css/typedom/color/CSSLab.h:
* Source/WebCore/css/typedom/color/CSSLab.idl:
* Source/WebCore/css/typedom/color/CSSOKLCH.cpp:
* Source/WebCore/css/typedom/color/CSSOKLCH.h:
* Source/WebCore/css/typedom/color/CSSOKLCH.idl:
* Source/WebCore/css/typedom/color/CSSOKLab.cpp:
* Source/WebCore/css/typedom/color/CSSOKLab.h:
* Source/WebCore/css/typedom/color/CSSOKLab.idl:
* Source/WebCore/css/typedom/color/CSSRGB.cpp:
* Source/WebCore/css/typedom/color/CSSRGB.h:
* Source/WebCore/css/typedom/color/CSSRGB.idl:
* Source/WebCore/css/typedom/numeric/CSSMathClamp.cpp:
* Source/WebCore/css/typedom/numeric/CSSMathClamp.h:
* Source/WebCore/css/typedom/numeric/CSSMathClamp.idl:
* Source/WebCore/css/typedom/numeric/CSSMathInvert.cpp:
* Source/WebCore/css/typedom/numeric/CSSMathInvert.h:
* Source/WebCore/css/typedom/numeric/CSSMathInvert.idl:
* Source/WebCore/css/typedom/numeric/CSSMathMax.cpp:
* Source/WebCore/css/typedom/numeric/CSSMathMax.h:
* Source/WebCore/css/typedom/numeric/CSSMathMax.idl:
* Source/WebCore/css/typedom/numeric/CSSMathMin.cpp:
* Source/WebCore/css/typedom/numeric/CSSMathMin.h:
* Source/WebCore/css/typedom/numeric/CSSMathMin.idl:
* Source/WebCore/css/typedom/numeric/CSSMathNegate.cpp:
* Source/WebCore/css/typedom/numeric/CSSMathNegate.h:
* Source/WebCore/css/typedom/numeric/CSSMathNegate.idl:
* Source/WebCore/css/typedom/numeric/CSSMathOperator.h:
* Source/WebCore/css/typedom/numeric/CSSMathOperator.idl:
* Source/WebCore/css/typedom/numeric/CSSMathProduct.cpp:
* Source/WebCore/css/typedom/numeric/CSSMathProduct.h:
* Source/WebCore/css/typedom/numeric/CSSMathProduct.idl:
* Source/WebCore/css/typedom/numeric/CSSMathSum.cpp:
* Source/WebCore/css/typedom/numeric/CSSMathSum.h:
* Source/WebCore/css/typedom/numeric/CSSMathSum.idl:
* Source/WebCore/css/typedom/numeric/CSSMathValue.h:
* Source/WebCore/css/typedom/numeric/CSSMathValue.idl:
* Source/WebCore/css/typedom/numeric/CSSNumericArray.cpp:
* Source/WebCore/css/typedom/numeric/CSSNumericArray.h:
* Source/WebCore/css/typedom/numeric/CSSNumericArray.idl:
* Source/WebCore/css/typedom/numeric/CSSNumericBaseType.h:
* Source/WebCore/css/typedom/numeric/CSSNumericBaseType.idl:
* Source/WebCore/css/typedom/numeric/CSSNumericType.cpp:
* Source/WebCore/css/typedom/numeric/CSSNumericType.h:
* Source/WebCore/css/typedom/numeric/CSSNumericType.idl:
* Source/WebCore/css/typedom/transform/CSSMatrixComponent.cpp:
* Source/WebCore/css/typedom/transform/CSSMatrixComponent.h:
* Source/WebCore/css/typedom/transform/CSSMatrixComponent.idl:
* Source/WebCore/css/typedom/transform/CSSMatrixComponentOptions.h:
* Source/WebCore/css/typedom/transform/CSSMatrixComponentOptions.idl:
* Source/WebCore/css/typedom/transform/CSSPerspective.cpp:
* Source/WebCore/css/typedom/transform/CSSPerspective.h:
* Source/WebCore/css/typedom/transform/CSSPerspective.idl:
* Source/WebCore/css/typedom/transform/CSSRotate.cpp:
* Source/WebCore/css/typedom/transform/CSSRotate.h:
* Source/WebCore/css/typedom/transform/CSSRotate.idl:
* Source/WebCore/css/typedom/transform/CSSScale.cpp:
* Source/WebCore/css/typedom/transform/CSSScale.h:
* Source/WebCore/css/typedom/transform/CSSScale.idl:
* Source/WebCore/css/typedom/transform/CSSSkew.cpp:
* Source/WebCore/css/typedom/transform/CSSSkew.h:
* Source/WebCore/css/typedom/transform/CSSSkew.idl:
* Source/WebCore/css/typedom/transform/CSSSkewX.cpp:
* Source/WebCore/css/typedom/transform/CSSSkewX.h:
* Source/WebCore/css/typedom/transform/CSSSkewX.idl:
* Source/WebCore/css/typedom/transform/CSSSkewY.cpp:
* Source/WebCore/css/typedom/transform/CSSSkewY.h:
* Source/WebCore/css/typedom/transform/CSSSkewY.idl:
* Source/WebCore/css/typedom/transform/CSSTransformComponent.cpp:
* Source/WebCore/css/typedom/transform/CSSTransformComponent.h:
* Source/WebCore/css/typedom/transform/CSSTransformComponent.idl:
* Source/WebCore/css/typedom/transform/CSSTransformValue.cpp:
* Source/WebCore/css/typedom/transform/CSSTransformValue.h:
* Source/WebCore/css/typedom/transform/CSSTransformValue.idl:
* Source/WebCore/css/typedom/transform/CSSTranslate.cpp:
* Source/WebCore/css/typedom/transform/CSSTranslate.h:
* Source/WebCore/css/typedom/transform/CSSTranslate.idl:
* Source/WebCore/dom/Element+ComputedStyleMap.idl:
* Source/WebCore/dom/Element.cpp:
(WebCore::Element::~Element):
* Source/WebCore/dom/Element.h:
* Source/WebCore/dom/ElementRareData.cpp:
* Source/WebCore/dom/ElementRareData.h:
(WebCore::ElementRareData::setComputedStyleMap):
(WebCore::ElementRareData::useTypes const):
* Source/WebCore/dom/StyledElement.cpp:
(WebCore::StyledElement::ensureAttributeStyleMap):
* Source/WebCore/dom/StyledElement.h:
* Source/WebCore/html/ImageBitmap.cpp:
(WebCore::ImageBitmap::createPromise):
* Source/WebCore/html/ImageBitmap.h:
* Source/WebCore/html/canvas/CanvasDrawImage.idl:
* Source/WebCore/html/canvas/CanvasFillStrokeStyles.idl:
* Source/WebCore/html/canvas/CanvasRenderingContext2DBase.cpp:
(WebCore::size):
(WebCore::CanvasRenderingContext2DBase::drawImage):
(WebCore::CanvasRenderingContext2DBase::createPattern):
* Source/WebCore/html/canvas/CanvasRenderingContext2DBase.h:
* Source/WebCore/inspector/InspectorCanvas.cpp:
(WebCore::InspectorCanvas::indexForData):
* Source/WebCore/inspector/InspectorCanvas.h:
* Source/WebCore/inspector/InspectorCanvasCallTracer.h:
* Source/WebCore/page/DeprecatedGlobalSettings.h:
(WebCore::DeprecatedGlobalSettings::cssTypedOMEnabled):
* Source/WebCore/page/WindowOrWorkerGlobalScope.idl:
* Source/WebKit/WebProcess/Inspector/WebInspectorUI.cpp:
(WebKit::WebInspectorUI::enableFrontendFeatures):
* Source/cmake/OptionsGTK.cmake:
* Source/cmake/OptionsMac.cmake:
* Source/cmake/OptionsPlayStation.cmake:
* Source/cmake/OptionsWPE.cmake:
* Source/cmake/OptionsWin.cmake:
* Source/cmake/WebKitFeatures.cmake:

Canonical link: https://commits.webkit.org/255862@main
